velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -238,6 +238,10 @@ class VectorFuzzer {
   // elements.
   RowVectorPtr fuzzInputRow(const RowTypePtr& rowType);
 
+  // Returns a RowVector whose children are flat vectors or a complex vectors
```

Comment:
Would you extract fuzzer changes into a separate PR?

Reply:
@mbasmanova I've filed another PR https://github.com/facebookincubator/velox/pull/6849 extracting the fuzzer changes. I will continue to use this PR to refactor `ParquetWriterTestBase.h`.